packages/lib/src/vis/heatmap/HeatmapVis.tsx

```diff
@@ -30,7 +30,7 @@ interface Props {
   aspect?: Aspect;
   showGrid?: boolean;
   title?: string;
-  dtype?: NumericType;
+  dtype?: NumericLikeType;
```

Comment:
Not a fan of `@h5web/lib` knowing about `NumericLikeType` (or any other `DType`). I'm thinking I might change this to `string` and move the call to `formatNumLikeType` into the mapped components at some point.